platform/darwin/src/MGLStyle.h

```diff
@@ -610,6 +610,18 @@ MGL_EXPORT
  */
 @property (nonatomic, strong) MGLLight *light;
 
+#pragma mark Managing Style's localization
```

Comment:
I’m not sure how we landed on “Managing” everything. 🤔 Could we just call this “Localization”?